src/utils.js

```diff
@@ -94,12 +94,36 @@ const emptyTransport = {
     warn: noop
 };
 
+function generateStackId() {
+    let id = '';
+    const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
You can put `chars` and `chars.length` into external variables so we do not instanciate a new array on each call

Reply:
ok